Kwenta/kwenta
# Added tally ho to rainbowkit

## Description
Add `Tally Ho` to the wallet list.
> Note that `Tally Ho` needs to be set as a default wallet in the setting.
![image](https://user-images.githubusercontent.com/4819006/191263238-1a7cc15f-a513-48e9-bf27-9ee3e4ac54b1.png)

## Related issue
#1402

## Motivation and Context


## How Has This Been Tested?
1. Open the page and click on the `connect wallet`
2. Select `Tally Ho` and confirm on the pop-up

## Screenshots (if appropriate):
![image](https://user-images.githubusercontent.com/4819006/191262912-7ee56d30-6fc0-4db1-a6f6-d394f292d072.png)
